### DISCUSSION TITLE
Include {date} in filename_format by default

### DISCUSSION THREAD

**Discussion by roweger** [DIGITALCRIMINALS/OnlyFans#317] [Ideas]
Including the `{date}` in the `"filename_format"` for the various sites in the config file by default would be extremely helpful. 

It seems most users don't think about this or are unaware of it, but having the published-date in the filename (particularly as `%Y-%m-%d`) automatically sorts the files so that images and videos from the same set aren't randomly scattered throughout the folder.

**Comment by DIGITALCRIMINAL** [comment]
I don't like anything other than the filename in the actual filename for SEO reasons.
